### ISSUE TITLE
训练开始没几步就报nan

### ISSUE THREAD

**Issue by maomaoyuchengzi** [WenmuZhou/PytorchOCR#293]
报错信息：
2024/06/04 03:23:10] torchocr INFO: run with torch 1.11.0 and device cuda:0
[2024/06/04 03:23:10] torchocr INFO: During the training process, after the 0th epoch, an evaluation is run every 10 epoch
[2024/06/04 03:23:25] torchocr INFO: epoch: [1/200], global_step: 10, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.023486, CTCLoss: nanxxx, NRTRLoss: nanxxx, loss: nanxxx, avg_reader_cost: 0.36281 s, avg_batch_cost: 1.48329 s, avg_samples: 100.1, ips: 67.48496 samples/s, eta: 27 days, 19:13:50
[2024/06/04 03:23:31] torchocr INFO: epoch: [1/200], global_step: 20, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000000, CTCLoss: nanxxx, NRTRLoss: nanxxx, loss: nanxxx, avg_reader_cost: 0.01449 s, avg_batch_cost: 0.53998 s, avg_samples: 93.8, ips: 173.70977 samples/s, eta: 18 days, 23:03:45
[2024/06/04 03:23:36] torchocr INFO: epoch: [1/200], global_step: 30, lr: 0.000001, acc: 0.000000, norm_edit_dis: 0.000000, CTCLoss: nanxxx, NRTRLoss: nanxxx, loss: nanxxx, avg_reader_cost: 0.01466 s, avg_batch_cost: 0.52133 s, avg_samples: 98.0, ips: 187.98018 samples/s, eta: 15 days, 21:32:32
参数配置信息：
Global:
  device: gpu
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 10
  output_dir: ./output/rec/rec_ppocr_v4_en_
  eval_epoch_step: [0, 10]
  cal_metric_during_train: true
  pretrained_model: /home/jovyan/xxx/ocr/PytorchOCR-dev/student.pth
  checkpoints:
  use_tensorboard: false
  infer_mode: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: &character_dict_path  /home/jovyan/pvc-ocr/ocr/mobile_ocr/rec/train_val_txt/20240603/all_label.txt
  max_text_length: &max_text_length 25
  use_space_char: &use_space_char true

Export:
  export_dir:
  export_shape: [ 1, 3, 48, 320 ]
  dynamic_axes: [ 0, 2, 3 ]

Optimizer:
  name: Adam #SGD
  lr: 0.001
  weight_decay: 3.0e-05

LRScheduler:
  name: CosineAnnealingLR
  warmup_epoch: 5

Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform:
  Backbone:
    name: PPLCNetV3
    scale: 0.95
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 120
            depth: 2
            hidden_dims: 120
            kernel_size: [1, 3]
            use_guide: True
      - NRTRHead:
          nrtr_dim: 384
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - NRTRLoss:

PostProcess:  
  name: CTCLabelDecode
  character_dict_path: *character_dict_path
  use_space_char: *use_space_char

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: MultiScaleDataSet
    ds_width: false
    data_dir: /home/jovyan/ocr-juicefs-readonly-pvc/xxx/
    ext_op_transform_idx: 1
    label_file_list:
    - /home/jovyan/pvc-ocr/ocr/mobile_ocr/rec/train_val_txt/20240603/train2.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  sampler:
    name: MultiScaleSampler
    scales: [[320, 32], [320, 48], [320, 64]]
    first_bs: &bs 128 #192
    fix_bs: false
    divided_factor: [8, 16] # w, h
    is_training: True
  loader:
    shuffle: true
    batch_size_per_card: *bs
    drop_last: true
    num_workers: 8
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /home/jovyan/ocr-juicefs-readonly-pvc/xxx/
    label_file_list:
    - /home/jovyan/pvc-ocr/ocr/mobile_ocr/rec/train_val_txt/20240603/val2.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 128
    num_workers: 4
同样的数据，paddleocr上训练不会出错。请教大佬原因。

**Comment by liwenju0** [comment]
I had the same problem and solved it by setting use_amp: True in Global
